### ISSUE TITLE
Links in JSON are unstyled

### ISSUE THREAD

**Issue by minimalsm** [ethereum/ethereum-org-website#8547]
### Describe the bug

The links that we create through HTML embedded in JSON are not getting styled. Other tags (e.g. bold, italics) seem to be working fine.

Some pages this effects:
https://ethereum.org/en/run-a-node/
https://ethereum.org/en/staking/
https://ethereum.org/en/layer-2/
https://ethereum.org/en/what-is-ethereum/

### To reproduce

1. Go to 'https://ethereum.org/en/layer-2/'
2. Search 'blockchain trilemma'
3. Hover over text

### Expected behavior

All links should be styled as per our theme

### Screenshots

<img width="780" alt="Screenshot 2022-11-09 at 11 46 11" src="https://user-images.githubusercontent.com/62268199/200827694-7b5ad305-150b-4c29-83b1-f9ebb6542c96.png">

### Additional context

First deploy I see this bug is 6.1.0 (#7766); it doesn't look to be present in 6.0.0 (#7658), which is when we started migrating to Chakra UI.

**Comment by TylerAPfledderer** [comment]
@minimalsm looks like the styling got lost when moving `GlobalStyle.tsx` to the Chakra theme in `styles.ts`. There may be a few others that of been missed as well?
